crates/turbopack-core/src/chunk/dev.rs

```diff
@@ -223,8 +223,14 @@ impl ChunkingContext for DevChunkingContext {
     }
 
     #[turbo_tasks::function]
-    fn chunk_list_path(self_vc: DevChunkingContextVc, ident: AssetIdentVc) -> FileSystemPathVc {
-        self_vc.chunk_path(ident.with_modifier(chunk_list_modifier()), ".json")
+    fn chunk_list_path(
+        self_vc: DevChunkingContextVc,
+        entry_chunk_path: FileSystemPathVc,
+    ) -> FileSystemPathVc {
+        self_vc.chunk_path(
+            AssetIdentVc::from_path(entry_chunk_path).with_modifier(chunk_list_modifier()),
+            ".json",
+        )
```

Comment:
Getting `[UpdateServer]: error invalid uri character` when trying to subscribe to the chunk list updates with:

```
{"type":"subscribe","path":"_next/static/chunks/[root of the server]__next_static_chunks_.env_12715c.js_e3a1b5._.json","headers":null}
```

Reply:
fixed